kubernetes/template/DeploymentTemplates/MainTemplate.json

```diff
@@ -147,6 +147,48 @@
         "description": "Network plugin which will deployed in Kubernetes cluster"
       },
       "type": "string"
+    },    
+    "windowsAdminUsername": {
+      "defaultValue": "azureuser",
+      "metadata": {
+        "description": "User name for the Windows virtual machines that are part of the Kubernetes cluster."
+      },
+      "type": "string"
+    },
+    "windowsAdminPassword": {
+      "defaultValue": "",
+      "metadata": {
+        "description": "User password for the Windows virtual machines that are part of the Kubernetes cluster."
+      },
+      "type": "securestring"
+    },
+    "windowsPackageBaseURL": {
+      "defaultValue": "",
+      "metadata": {
+        "description": "URL path from where Windows binaries will be downloaded."
+      },
+      "type": "string"
+    },
+    "windowsPackageName": {
+      "defaultValue": "",
+      "metadata": {
+        "description": "Name of the Windows package which will be installed."
+      },
+      "type": "string"
+    },
+    "windowsAgentPoolProfileCount": {
+      "defaultValue": "0",      
+      "metadata": {
+        "description": "Kubernetes Windows node pool profile count"
+      },
+      "type": "string"
+    },
+    "windowsAgentPoolProfileVMSize": {
+      "defaultValue": "Standard_D2_v2",
+      "metadata": {
+        "description": "The VMSize of Kubernetes Windows node VMs"
```

Comment:
I am unsure about a few descriptions, this one maybe better like this: `The virtual machine size of the Kubernets Windows nodes`

Reply:
updated.